contracts/src/components/road.cairo

```diff
@@ -42,18 +32,6 @@ impl RoadImpl of RoadTrait {
         road
     }
 
-
-    #[inline(always)]
-    fn speed_boost(self: Road) -> u64 {
-        2   
-    }
-
-
-    #[inline(always)]
-    fn travel(ref self: Road, world: IWorldDispatcher) {
```

Comment:
This was removed because i think after we clean out the codebase, `usage_count ` would only need to be updated in the travel system or a related one.